applications/StructuralMechanicsApplication/tests/pendulus_test/pendulus_TL_test.mdpa

```diff
@@ -0,0 +1,307 @@
+Begin ModelPartData
```

Comment:
Why so many elements? Can t you make the test smaller?

Reply:
For the pendulus, is already quite small

Reply:
Well...you could use 5-6 elements instead of 50...